crates/oxc_semantic/src/lib.rs

```diff
@@ -22,9 +25,13 @@ pub struct Semantic<'a> {
 
     scopes: ScopeTree,
 
+    symbols: SymbolTable,
+
     trivias: Rc<Trivias>,
 
     module_record: ModuleRecord,
+
+    resolved_references: FxHashMap<AstNodeId, ResolvedReference>,
```

Comment:
Should this be part of the symbol table? The symbol table should be an abstract for all symbol related code.

Reply:
Done.